src/assets/TokenBalancesController.test.ts

```diff
@@ -1,18 +1,16 @@
 import { createSandbox, stub } from 'sinon';
 import { BN } from 'ethereumjs-util';
-import HttpProvider from 'ethjs-provider-http';
+// import HttpProvider from 'ethjs-provider-http';
```

Comment:
This comment can be removed. I see that it was only used for the `MAINNET_PROVIDER`, which was itself removed.

```suggestion
```